### PR TITLE
gh-91968: socket SO_USER_COOKIE/SO_RTABLE from BSD.

### PR DESCRIPTION
**Pull request by devnexen**
Those are somewhat equivalent to Linux' SO_MARK.

### CODE REVIEW

**Comment by devnexen** [comment]
@JelleZijlstra any chance ? :-P

**Comment by JelleZijlstra** [comment]
Too late for 3.11, we're in feature freeze. I'll review it for 3.12 though.

File: Doc/library/socket.rst [hunk 1]
```diff
@@ -392,6 +392,11 @@ Constants
       Added ``TCP_CONNECTION_INFO``. On MacOS this constant can be used in the
       same way that ``TCP_INFO`` is used on Linux and BSD.
 
+   .. versionchanged:: 3.11
```
Comment (JelleZijlstra) [inline Doc/library/socket.rst]:
```suggestion
   .. versionchanged:: 3.12
```

File: Doc/library/socket.rst [hunk 2]
```diff
@@ -392,6 +392,11 @@ Constants
...
+      Added ``SO_RTABLE`` and ``SO_USER_COOKIE``. Respectively for OpenBSD
+      and FreeBSD those constants can be used in the same way that ``SO_MARK``
```
Comment (JelleZijlstra) [inline Doc/library/socket.rst]:
```suggestion
      Added ``SO_RTABLE`` and ``SO_USER_COOKIE``. On OpenBSD
      and FreeBSD respectively those constants can be used in the same way that ``SO_MARK``
```
More consistent with wording in previous entries.

**Review by JelleZijlstra — APPROVED** [review]

**Pull request merged**